packages/modelviewer.dev/index.html

```diff
@@ -500,6 +530,14 @@ <h3 class="grouping-title">Methods</h3>
               this method is invoked, but the model may not update until the next
               render frame.</p>
             </li>
+            <li>
+              <div>setURLModifier(callback)</div>
```

Comment:
Let's make this method static (it seems as though it is already effectively static in terms of side-effects, and the last `<model-viewer>` configured will win).

Also, let's take this opportunity to pick a more concise name. I'll suggest: `mapURLs(...)`.